docs/source/app/holography.rst

```diff
@@ -0,0 +1,195 @@
+Off-axis electron holography
```

Comment:
Above this, you can add a label

```
.. _`holography`:
```

and link to it using :ref:`holography` in the text and docstrings.